### DISCUSSION TITLE
How can I marshal PostgreSQL JSONB type back to Java Map

### DISCUSSION THREAD

**Discussion by d3bt3ch** [jdbi/jdbi#2679] [Q&A] [answered]
We have a situtaion where we are storing a java map in PostgreSQL jsonb column. How can we read it back in java. We are using PostgreSQL function with out parameter as JSONB.

**Comment by stevenschlansker** [comment, marked as answer]
Hi @d3btech , thanks for pointing this out. I think it should be possible using `registerOutParameter` with a custom `CallableStatementMapper`, but this should be easier and more obvious to do "out of the box". I'll convert this to a feature request to make this easier to do.

**Reply by d3bt3ch** [reply]
@stevenschlansker Thanks for putting this as a feature request. For now, we created a `CallableStatementMapper` and is working like a charm.

**Reply by d3bt3ch** [reply]
@stevenschlansker Has this feature been implemented ?
